certora/README.md

```diff
@@ -0,0 +1,51 @@
+# Certora 
+
+This folder provides a fully functional skeleton for integrating with Certora.
+There are two configurations:
+
+
+
+## Run From the Command Line
+
+To run the checks from the command line, first install the Python SDK:
+`pip install certora-cli`
```

Comment:
that's often not enough and we need to add the certora package to PATH. At least according to the last time i reinstalled

Reply:
maybe add link to full installation guide

Reply:
done